sdk/loadtestservice/ci.yml

```diff
@@ -29,5 +29,5 @@ extends:
   parameters:
     ServiceDirectory: loadtestservice
     Artifacts:
-    - name: azure-mgmt-loadtestservice
-      safeName: azuremgmtloadtestservice
+    - name: azure-developer-loadtesting
+      safeName: azuredeveloperloadtesting
```

Comment:
you could add new line but you can't change the existing lines which has been published